src/wazuh_modules/wm_syscollector.c

```diff
@@ -151,6 +152,15 @@ void* wm_sys_main(wm_sys_t *sys) {
         syscollector_start_ptr = so_get_function_sym(syscollector_module, "syscollector_start");
         syscollector_stop_ptr = so_get_function_sym(syscollector_module, "syscollector_stop");
         syscollector_sync_message_ptr = so_get_function_sym(syscollector_module, "syscollector_sync_message");
+
+        void* rsync_module = NULL;
+        if(rsync_module = so_check_module_loaded("rsync"), rsync_module) {
+            void (*rsync_initialize_ptr)(full_log_fnc_t) = so_get_function_sym(rsync_module, "rsync_initialize_log_function");
```

Comment:
I would rename this as `void (*rsync_init_logging_ptr)(full_log_fnc_t) = so_get_function_sym(rsync_module, "rsync_init_logging");` to match the definition convention followed above for syscollector ptrs.

Reply:
Like we already have the `rsync_initialize()` method, I selected `rsync_initialize_log_function()`.

To follow the convention, I'm writing

```c++
void (*rsync_initialize_log_function_ptr)(full_log_fnc_t) = so_get_function_sym(rsync_module, "rsync_initialize_log_function");
```

What do you think?